orttraining/orttraining/python/training/ortmodule/_training_manager.py

```diff
@@ -100,6 +103,26 @@ def forward(self, *inputs, **kwargs):
                 if self._loglevel <= _logger.LogLevel.WARNING:
                     warnings.warn("Fast path enabled - skipping checks for rebuilding gradient graph, execution agent creation, and device during training.",
                                   UserWarning)
+        # create cache object
+        if self._enable_grad_acc_optimization:
+            if self._cache == None:
+                self._cache = C.OrtValueCache()
+
+            # parse param versions to detect change or no change
+            if self._param_version_map == None:
+                self._param_version_map = dict()
```

Comment:
Similarly to the above, could this be initialized at `GraphExecutionManager` as `dict()` as opposed to `None` and prevent this second initialization?